src/pg_to_evalscript/utils.py

```diff
@@ -3,7 +3,7 @@
 
 def list_supported_processes():
     process_definitions_directory = "javascript_processes"
-    implicitly_supported_processes = ["load_collection", "save_result", "reduce_dimension", "apply"]
+    implicitly_supported_processes = ["load_collection", "save_result", "reduce_dimension", "apply", "array_apply"]
```

Comment:
Not connected to this MR, but to https://github.com/openEOPlatform/openeo-pg-evalscript-converter/pull/36

I think we forgot to add the `count` process.

FFTR.

Reply:
I think this should **not** be added, since the definition of the `count` process is in the file and thus gets read from there when running this test.

Reply:
Ah thanks, true. Got carried away when I was checking `node.py`. Resolving

Reply:
This is correct, however the way `reduce_dimension` and `apply` process are implemented isn't great and will probably be moved to separate files and implemented in a similar manner like `count`. It would be great if this was done here as well, because we can then add unit tests as well.

Reply:
Makes sense yes, will move it to a file and define some unit tests